hello_world.iml

```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
+  <component name="CheckStyle-IDEA-Module">
+    <option name="configuration">
+      <map />
+    </option>
+  </component>
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
     <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
```

Comment:
Why get rid of `/main/java` in the source folder url ?

Reply:
I converted the project over to maven. IntelliJ is no longer handing the building of the Java project, so does not need this information. Maven requires a certain file layout, which is now in place. This defines the directory layout: https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html